init.sh

```diff
@@ -22,17 +22,19 @@ done
 if test -z "$machine" ; then
    echo "An invalid target name was given of $choice, available targets are:"
    printf '%s\n' "${targets[@]}"
-   unset machine choice
+   unset machine choice eula
+   echo "Usage: source init.sh target [accept-eula] [-f]"
```

Comment:
I'd find it difficult to say what parameters there were and in which order without specifying them all (though I can see a reason for omitting the source init.sh bit)